examples/index.json

```diff
@@ -989,7 +989,7 @@
         "category": "Language concepts",
         "samples": [
             {
-                "name": "Assertions",
+                "name": "assertions",
```

Comment:
Why are we starting with a lower case letter?

Reply:
Sorry, that was a mistake. Fixed.